ckanext/harvest/logic/action/update.py

```diff
@@ -491,6 +496,88 @@ def harvest_jobs_run(context, data_dict):
 
     return []  # merely for backwards compatibility
 
+def send_error_mail(context, job_obj):
+
+    sql = 'select name from package where id = :source_id;'
+
+    model = context['model']
+
+    q = model.Session.execute(sql, {'source_id': job_obj.source.id})
+
+    for row in q:
+        harvest_name = str(row['name'])
+
+    ckan_site_url = config.get('ckan.site_url')
+    job_url = ckan_site_url + '/harvest/' + harvest_name + '/job/' + job_obj.id
+
+    msg = 'This is a failure-notification of the latest harvest job (' + job_url + ') set-up in ' + ckan_site_url + '.\n\n'
+
+    sql = '''select g.title as org, s.title as job_title from member m
+            join public.group g on m.group_id = g.id
+            join harvest_source s on s.id = m.table_id
+            where table_id = :source_id;'''
+
+    q = model.Session.execute(sql, {'source_id': job_obj.source.id})
+
+    for row in q:
+        msg += 'Organization: ' + str(row['org']) + '\n\n'
+        msg += 'Harvest Job Title: ' + str(row['job_title']) + '\n\n'
+
+    msg += 'Date of Harvest: ' + str(job_obj.created) + ' GMT\n\n'
```

Comment:
Harvest job `created` is created as an UTC date, better to use this than GMT to avoid confusion, even if they are nearly always the same.